### PR TITLE
Add a debug option to bypass MCT facade

### PR DESCRIPTION
**Pull request by ofirgo**
## Pull Request Description:
If MCT is set with `bypass` in the DebugConfig, it returns the input model without any modifications.

In addition, minor hierarchy fix in common unit tests

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tests_pytest/keras_tests/_keras_base_tests/base_keras_mct_bypass.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# ==============================================================================
+
+import keras
+import numpy as np
+from tests_pytest._fw_tests_common_base.base_mct_bypass_test import BaseMCTBypassTest
+
+
+class BaseKerasMCTBypassTest(BaseMCTBypassTest):
+
+
```
Comment (lior-dikstein) [inline tests_pytest/keras_tests/_keras_base_tests/base_keras_mct_bypass.py:23]: Can you add a short description of what the test does?
This test class is designed to verify that a PyTorch model, when processed through the MCT with a bypass enabled, retains its original architecture and parameters.

File: tests_pytest/keras_tests/e2e_tests/test_mct_bypass.py [hunk 2]
```diff
@@ -12,3 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+from model_compression_toolkit.gptq import keras_gradient_post_training_quantization
+from model_compression_toolkit.ptq import keras_post_training_quantization
+from tests_pytest.keras_tests._keras_base_tests.base_keras_mct_bypass import BaseKerasMCTBypassTest
+
+
+
+class TestKerasPTQMCTBypass(BaseKerasMCTBypassTest):
```
Comment (lior-dikstein) [inline tests_pytest/keras_tests/e2e_tests/test_mct_bypass.py:21]: Why implement something like this and not just add the API function as a parameter fixture?

**Review by lior-dikstein — APPROVED** [review]
